nh_eobs_mental_health/models/nh_clinical_pme_obs_stop.py

```diff
@@ -139,7 +139,7 @@ def create_new_ews(self):
         activity_obs_stop = self.get_activity()
         activity_spell = self.spell.get_activity()
 
-        new_ews_id = ews_model.create_activity(
+        new_ews_activity_id = ews_model.create_activity(
```

Comment:
I wonder if it's worth splitting this method into two methods `create_new_ews` and `schedule_new_ews` to represent the two different but related jobs?